src/PollRow.js

```diff
@@ -6,7 +6,7 @@ export default class {
         this._votes = votes;
         this._coefficient = coefficient;
         this._effectiveVotes = Math.ceil(this._votes * this._coefficient);
-        this._node = $(`<div>${title} (ev=${this._effectiveVotes};c=${this._coefficient};ov=${this._votes})</div>`);
+        this._node = $(`<div><h2>${title} <b>${this._effectiveVotes}</b></h2><br /> Coefficient ${this._coefficient} Actual Votes${this._votes})</div>`;
```

Comment:
- Close the parens a the end of the expression
- Don't use `<br>`. Use block elements and CSS for positioning.